### PR TITLE
[mobile]add QUIC idle timeout setting to EngineBuilder

### PR DESCRIPTION
**Pull request by RenjieTang**
Commit Message: add QUIC idle timeout setting to EngineBuilder
Risk Level: Low
Testing: n/a
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: mobile only

### CODE REVIEW

File: mobile/library/cc/engine_builder.cc [hunk 1]
```diff
@@ -267,6 +267,11 @@ EngineBuilder& EngineBuilder::setUpstreamTlsSni(std::string sni) {
   return *this;
 }
 
+EngineBuilder&
+EngineBuilder::setQuicConnectionIdleTimeoutSeconds(int quic_connection_idle_timeout_seconds) {
+  quic_connection_idle_timeout_seconds_ = quic_connection_idle_timeout_seconds;
```
Comment (fredyw) [inline mobile/library/cc/engine_builder.cc:272]: need to `return *this`

**Review by fredyw — CHANGES_REQUESTED** [review]
Do we need to update the Cronvoy API as well?

**Comment by RenjieTang** [comment]
> Do we need to update the Cronvoy API as well?

We eventually want to do it for Android. But I think we probably will do it on the Android native integration.

**Review by fredyw — APPROVED** [review]

**Pull request merged**